app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_results = {}
+
+    def wrapper(*args, **kwargs) -> dict:
```

Comment:
does this function return `dict`?

Reply:
seems doesn't
I checked "Readme" and it looks like "int"